addons/config/configs/CfgWeapons.hpp

```diff
@@ -512,6 +512,24 @@ class CfgWeapons
         };
     };
 
+    class DMR_02_base_F;
+    class srifle_DMR_02_F : DMR_02_base_F // Noreen "Bad News" ULR
+    {
+        magazines[] = {"TB_mag_10Rnd_338_LS_Tracer"}; // {"10Rnd_338_Mag"}
+        magazineWell[] = {"MAR10_338","TB_magwell_10Rnd_338"}; // {"MAR10_338"}
```

Comment:
Wenn du nur etwas hinzufügen willst wäre es:
```suggestion
        magazineWell[] += {"TB_magwell_10Rnd_338"}; // {"MAR10_338"}
```

Wenn es allerdings alles überschreiben soll, dann so.